dockerfiles/Dockerfile-mina-rosetta

```diff
@@ -89,8 +88,17 @@ RUN export GOBIN="$(pwd)/bin" \
   && /usr/lib/go/bin/go mod tidy \
   && /usr/lib/go/bin/go install
 
+# Rosetta-demo artifacts / build dir removal:
```

Comment:
Is this documentation for running the test locally? It would be better to move this to somewhere more discoverable (or just delete it).